packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -243,6 +243,16 @@ bool shouldDisableTab(String key) {
   return qsParams['hide']?.split(',')?.contains(key) ?? false;
 }
 
+bool get useDarkTheme {
```

Comment:
This already exists.
See
bool get isDarkTheme => _isDarkTheme;

in theme.dart
Go ahead and remove this code and use that code.

Reply:
Whoops, good spot. Fixed!